internal/datacoord/import_job.go

```diff
@@ -91,15 +95,36 @@ type ImportJob interface {
 	GetCompleteTime() string
 	GetFiles() []*internalpb.ImportFile
 	GetOptions() []*commonpb.KeyValuePair
+	Record(stage string, msg string, elapse bool)
 	Clone() ImportJob
 }
 
 type importJob struct {
 	*datapb.ImportJob
+
+	hasRecorded map[string]bool
```

Comment:
add index state to task, and get rid of this hasRecorded.

Reply:
removed